.github/workflows/acceptance-py3.6.yml

```diff
@@ -0,0 +1,108 @@
+    runs-on: ubuntu-16.04
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        instance: [0, 1, 2]
+
+    env:
+      MIGRATIONS_TEST_MIGRATE: 1
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: volta-cli/action@v1
+
+      # If we make these jobs "required" to merge on GH, then on every PR, GitHub automatically
+      # creates a status check in the "pending" state. This means that the workflow needs to run
+      # for every PR in order to update the status checks.
+      #
+      # In order to optimize CI usage, we want the tests to only run when python files change,
+      # since frontend changes should have no effect on these test suites. We cannot use GH workflow
+      # path filters because entire workflow would be skipped vs skipping individual jobs which
+      # would still allow this status check to pass.
+      - name: Check for python file changes
+        uses: getsentry/paths-filter@v2
+        id: changes
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+      # XXX: If taking snapshots with this, be sure to remove above and the following `if` conditions!
+
+      - name: Set python version output
+        id: python-version
+        if: steps.changes.outputs.backend == 'true'
+        run: |
+          echo "::set-output name=python-version::$(awk 'FNR == 2' .python-version)"
+
+      # Until GH composite actions can use `uses`, we need to setup python here
+      - uses: actions/setup-python@v2
+        if: steps.changes.outputs.backend == 'true'
+        with:
+          python-version: ${{ steps.python-version.outputs.python-version }}
+
+      - name: Setup pip
+        uses: ./.github/actions/setup-pip
+        id: pip
+        if: steps.changes.outputs.backend == 'true'
+
+      - name: pip cache
+        uses: actions/cache@v2
+        if: steps.changes.outputs.backend == 'true'
+        with:
+          path: ${{ steps.pip.outputs.pip-cache-dir }}
+          key: ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ hashFiles('**/requirements-*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}
+
+      - name: Setup sentry python env
+        uses: ./.github/actions/setup-sentry
+        id: setup
+        if: steps.changes.outputs.backend == 'true'
+        with:
+          python: 3
+          snuba: true
+
+      - name: yarn cache
+        uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        if: steps.changes.outputs.backend == 'true'
+        with:
+          path: ${{ steps.setup.outputs.yarn-cache-dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install Javascript Dependencies
+        if: steps.changes.outputs.backend == 'true'
```

Comment:
@BYK The intention is to skip tests completely if there are no backend changes (as frontend changes should be covered by the 2.7 acceptance tests).

Re: cache - we're not caching `node_modules`, so we'd still have to run `yarn`

Reply:
>@BYK The intention is to skip tests completely if there are no backend changes (as frontend changes should be covered by the 2.7 acceptance tests).

This might be worth a comment here as it is not very obvious (to me at least).

>Re: cache - we're not caching node_modules, so we'd still have to run yarn

Wow, why not tho?

Reply:
There is a generic comment about this a bit above https://github.com/getsentry/sentry/pull/21386/files#diff-88f3cbb95a8fc264e7646e08b88c19a75b3f8176c8ed5fbab30f3ceeb6778bf2R27-R34

re: node modules, we cache yarn's global cache, caching `node_modules` itself needs some more testing/measurements - my initial tests w/ it had varying results as we have to upload/download the blackhole that is `node_modules`

Reply:
>re: node modules, we cache yarn's global cache, caching node_modules itself needs some more testing/measurements - my initial tests w/ it had varying results as we have to upload/download the blackhole that is node_modules

If that's the case, maybe we can experiment with PnP or Yarn v2?